clients/tv/.eslintrc.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": [
+    "airbnb",
+    "react-app"
```

Comment:
Teoretycznie możesz użyć mojego configa, tam mam kilka rzeczy pod TS ugrane już:
https://github.com/Deseteral/eslint-config/blob/master/typescript.js

Reply:
I to by dało jedną zależność tylko

Reply:
wygląda na to, że z CRA jest jakiś problem, merguję ten PR i robię custom setup do webpacka, tak będzie najkorzystniej